clients/src/test/java/org/apache/kafka/clients/consumer/internals/MembershipManagerImplTest.java

```diff
@@ -2457,10 +2538,7 @@ private CompletableFuture<Void> mockRevocationNoCallbacks(boolean withAutoCommit
         doNothing().when(subscriptionState).markPendingRevocation(anySet());
         when(subscriptionState.rebalanceListener()).thenReturn(Optional.empty()).thenReturn(Optional.empty());
         if (withAutoCommit) {
-            when(commitRequestManager.autoCommitEnabled()).thenReturn(true);
-            CompletableFuture<Void> commitResult = new CompletableFuture<>();
-            when(commitRequestManager.maybeAutoCommitSyncBeforeRevocation(anyLong())).thenReturn(commitResult);
-            return commitResult;
```

Comment:
uhm removing this will make that in all tests that want to block on commit we will have to call the same `when(commitRequestManager.maybeAutoCommitSyncBeforeRevocation...` we had here I guess? I do see that you had to add this line in many places in the end. Again, seems related to how we're using an insntace of commitRequestManager mentioned in comments above. (and if we are able to move to a mocked commitManager this should probably stay?)

Reply:
I have reverted changes here